clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -59,7 +62,7 @@ def reload_state(self) -> None:
         self._pods_by_ip = self._get_pods_by_ip()
 
     def get_resource_pending(self, resource_name: str) -> float:
-        return getattr(allocated_node_resources(self.get_unschedulable_pods()), resource_name)
+        return getattr(allocated_node_resources([p for p, __ in self.get_unschedulable_pods()]), resource_name)
```

Comment:
So this is left over from a (probably-outdated) entry in the Yelp style guide that you should use __ for unnamed variables because _ is used for string translations... _most_ other uses in Clusterman follow this pattern, which I could be argued into changing but kinda want to leave it for now?  Unless you feel strongly.